specification/loadtestservice/LoadTestService.Management/tspconfig.yaml

```diff
@@ -25,8 +29,28 @@ options:
     package-details:
       name: "@azure/arm-loadtesting"
   "@azure-tools/typespec-python":
-    package-dir: "azure-mgmt-loadtesting"
+    package-name: "azure-mgmt-loadtesting"
```

Comment:
you should use `package-dir`. refer the doc: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml#L19

Reply:
Done